.github/workflows/install.yml

```diff
@@ -4,15 +4,15 @@ name: Install from scratch
 on:
   schedule:
     # Run at 7am every day
-    # - cron:  '0 7 * * *'
+    - cron:  '0 7 * * *'
```

Comment:
If this is going to cost a significant amount of money, let's talk!

Reply:
No, I don't expect it is.